src/flags.ml

```diff
@@ -117,25 +117,28 @@ module Smt = struct
 
   (* Active SMT solver. *)
   type solver = [
+    | `Boolector_SMTLIB
     | `Z3_SMTLIB
     | `CVC4_SMTLIB
     | `Yices_SMTLIB
     | `Yices_native
     | `detect
   ]
   let solver_of_string = function
+    | "Boolector" -> `Boolector_SMTLIB
     | "Z3" -> `Z3_SMTLIB
     | "CVC4" -> `CVC4_SMTLIB
     | "Yices2" -> `Yices_SMTLIB
     | "Yices" -> `Yices_native
     | _ -> Arg.Bad "Bad value for --smt_solver" |> raise
   let string_of_solver = function
+    | `Boolector_SMTLIB -> "Boolector"
     | `Z3_SMTLIB -> "Z3"
     | `CVC4_SMTLIB -> "CVC4"
     | `Yices_SMTLIB -> "Yices2"
     | `Yices_native -> "Yices"
     | `detect -> "detect"
-  let solver_values = "Z3, CVC4, Yices, Yices2"
+  let solver_values = "Boolector, Z3, CVC4, Yices, Yices2"
```

Comment:
Use the order of detection (Z3, Yices2, CVC4, Yices, Boolector)

Reply:
Done!